eve/workers/pod-builder/pod.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
We should probably make a decision on `.yaml` vs. `.yml`.

As it stands, we have `salt/metalk8s/defaults.yaml` and `eve/workers/pod-builder/pod.yml`. Since your PR is the one in review, I'm tempted to say we go for `.yaml` because you're able to make the changes, otherwise it's potato vs. potato.

Reply:
I think `yaml` is the one used with the kubernetes community, so maybe stick with that ;)